typescript-declarations/@types/carp.core-kotlin-carp.deployments.core/index.d.ts

```diff
@@ -38,25 +38,25 @@ declare module 'carp.core-kotlin-carp.deployments.core'
             abstract class NotDeployed
```

Comment:
```suggestion
            abstract class NotDeployed extends DeviceDeploymentStatus
```
This is something we could also fix with this PR, otherwise I'll add it in a later commit.

Reply:
Thanks! But, I prefer to keep commits cohesive. I suggest you simply change this as part of a relevant commit on your future PR.